src/lib.rs

```diff
@@ -65,23 +65,58 @@ pub mod probes {
     pub fn transaction__done(conn_id: Uuid, depth: i64, committed: u8) {}
 }
 
-/// A [`Connection`] that includes DTrace probe points.
+/// A [`Connection`] wrapper that inserts DTrace probe points.
```

Comment:
do you want to refer to the crate-level docs rather than repeating?

Reply:
Done in 0358fa4